bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestCollections.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Diagnostics;
+using NUnit.Framework;
+
+namespace ElectionGuard.Encrypt.Tests
+{
+    [TestFixture]
+    public class TestCollections
+    {
+        [Test]
+        public void Test_Can_Create_Linked_list()
+        {
+            // Arrange
+            var list = new LinkedList();
+
+            // Act
+            list.Append("some", "value");
+            list.Append("another", "thing");
+
+            var firstValue = list.GetValueAt(0);
+            var secondValue = list.GetValueAt(1);
+
+            // Assert
+            Assert.That(firstValue == "value");
```

Comment:
Love these small, clean, and simple tests.  Quick tip: if you `Assert.Equals(firstValue, "value")`, then when the unit test fails it will provide a more helpful error message like: "firstValue was expected to be value but was actually thing".  Alternately if you feel like adopting Shouldly you can write it like `firstValue.ShouldBe("value")`, of which I'm a big fan, it has a ton of nice helpers, and is especially nice for collections (e.g. `var theItem = collection.ShouldContainSingleItem()`)..

This comment is applicable to several places in this PR, I'll just leave it as this one comment.